### PR TITLE
Incorrect text for settlement status open

### PR DESCRIPTION
**Pull request by himanshu007-creator**
## Description

Fixes an existing bug

## Motivation and context

Why is this change required? What problem does it solve?

#130 
## How has this been tested?

Please describe in detail how you tested your changes.

Include details of your testing environment, and the tests you ran to
see how your change affects other areas of the code, etc.

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

Go over all the following points, and put an `x` in all the boxes that apply.

Please, please, please, don't send your pull request until all of the boxes are ticked. Once your pull request is created, it will trigger a build on our continuous integration server to make sure your [tests and code style pass](https://help.github.com/articles/about-required-status-checks/).

- [x] I have read the **[CONTRIBUTING](CONTRIBUTING.md)** document.
- [x] My pull request addresses exactly one patch/feature.
- [x] I have created a branch for this patch/feature.
- [x] Each individual commit in the pull request is meaningful.
- [ ] I have added tests to cover my changes.
- [ ] If my change requires a change to the documentation, I have updated it accordingly.

If you're unsure about any of these, don't hesitate to ask. We're here to help!

### CODE REVIEW

File: mollie/settlements.go [hunk 1]
```diff
@@ -13,11 +13,11 @@ import (
...
-type SettlementStatus string
+type SettlementStatus
 
 // Available settlement statuses
 const (
-	SettlementStatusOpen    = "string"
+	SettlementStatusOpen    = "open"
 	SettlementStatusPending = "pending"
 	SettlementStatusPaidOut = "paidout"
 	SettlementStatusFailed  = "failed"
```
Comment (VictorAvelar) [inline mollie/settlements.go:23]: These constants should be typed as `SettlementStatus` string alias.
```suggestion
	SettlementStatusOpen      SettlementStatus = "open"
	SettlementStatusPending  SettlementStatus = "pending"
	SettlementStatusPaidOut  SettlementStatus = "paidout"
	SettlementStatusFailed     SettlementStatus = "failed"
```

**Review by VictorAvelar — CHANGES_REQUESTED** [review]
Thanks for the PR, make sure to have a look at the commit suggestions I left, I will be happy to clarify any further question.

Also as a side note, please make sure to gofmt the files once the PR is updated.

Thanks 🚀

**Review by VictorAvelar — APPROVED** [review]

**Comment by VictorAvelar** [comment]
@himanshu007-creator Thanks for your contribution!

**Comment by himanshu007-creator** [comment]
> @himanshu007-creator Thanks for your contribution!

Thank YOU!🙂

**Pull request merged**